Apps/Database/src/Models/UserFeedback.cs

```diff
@@ -32,5 +32,10 @@ public class UserFeedback : AuditableEntity
 
         [MaxLength(500)]
         public string? Comment { get; set; }
+
+        [MaxLength(52)]
+        public string? UserProfileId { get; set; }
```

Comment:
The isSatisfied field is no longer needed.

Reply:
+1

Reply:
Correct but it exists in the DB and I think that any cleanup of the DB should be done after the code is delivered.  Otherwise, you can't really rollback.